Cognite.Simulator.Tests/UtilsTests/ModelLibraryTest.cs

```diff
@@ -208,7 +209,7 @@ await lib.GetRunTasks(linkedTokenSource.Token)
                     Assert.Equal(SimulatorModelRevisionStatus.failure, modelInState.ParsingInfo.Status);
                     Assert.True(modelInState.ParsingInfo.Error);
                     // last updated time should not change
-                    Assert.Equal(revision.LastUpdatedTime, modelInState.ParsingInfo.LastUpdatedTime);
+                    Assert.Equal(revision.LastUpdatedTime, modelInState.UpdatedTime);
```

Comment:
Can we reduce the `LargeFileSize` externally to something very small like 1MB and test that code with a generated file?

Reply:
it's a bit of work, but I'll try to see what we can do with a fresh eyes on Monday then

Reply:
looks good